packages/functional-tests/tests/settings/deleteAccount.spec.ts

```diff
@@ -33,7 +33,7 @@ test.describe('severity-1 #smoke', () => {
     // Enter correct password
     await deleteAccount.setPassword(credentials.password);
     await deleteAccount.submit();
-    const success = await page.waitForSelector('.success');
+    const success = page.locator('.success');
     // TODO: "Error: toBeVisible can be only used with Locator object"
     // eslint-disable-next-line playwright/prefer-web-first-assertions
     expect(await success.isVisible()).toBeTruthy();
```

Comment:
Sorry, I know this is draft, but for my future reviewing self...
I think we'll need to do some grepping and converting the asserts into something like this, and then removing the TODO comments and `eslint-disable-next-line` stuff:
```js
expect(success).toBeVisible();
```